config.yaml

```diff
@@ -3,4 +3,4 @@ database:
   user: "postgres"
   dbname: "teaching_platform"
   port: 5432
-  password: "Rust098/*-"
\ No newline at end of file
+  password: "-"
```

Comment:
@Czupryn013 you might want to consider `dotenv`: https://www.nylas.com/blog/making-use-of-environment-variables-in-python/